sheepdog/transactions/upload/entity.py

```diff
@@ -623,33 +630,6 @@ def register_index(self):
             record=alias, hashes=hashes, size=size, release='private'
         )
 
-    def flush_to_session(self):
```

Comment:
That was a function redefininion, moved diff to the original function definition